Resources/Private/Fusion/Prototypes/Checkbox.fusion

```diff
@@ -1,25 +1,33 @@
 prototype(Neos.Fusion.Form:Checkbox)  < prototype(Neos.Fusion.Form:FieldComponent) {
-    @propTypes {
-        checked = ${PropTypes.boolean}
-    }
 
-    checked = false
+    renderer = Neos.Fusion:Component {
+
+        #
+        # pass down props
+        #
+        attributes = ${props.attributes}
+        content = ${props.content}
```

Comment:
Explanation why we pass props.content down, despite not using it below (Backend Module FieldContainer e.g.)

Reply:
Removed passing those down as all deriving prototyoes should at least @process the outer renderer.